server/platform/services/sharedchannel/util.go

```diff
@@ -148,3 +148,10 @@ func reducePostsSliceInCache(posts []*model.Post, cache map[string]*model.Post)
 	}
 	return reduced
 }
+
+func SafeString(p *string) string {
```

Comment:
I'm not in love with these sorts of wrappers, but I understand it can get verbose.